lms/templates/verify_student/reverify_not_allowed.html

```diff
@@ -0,0 +1,28 @@
+<%!
+    from django.utils.translation import ugettext as _
+    from django.core.urlresolvers import reverse
+%>
+
+<%inherit file="../main.html" />
+<%block name="pagetitle">${_("Re-Verification")}</%block>
+
+<%block name="content">
+<div class="reverify-blocked">
+    <h1 class="title">${_("Reverification")}</h1>
+
+    <div class="instructions">
+        <p>
+            % if status in ["pending", "approved"]:
+                ${_("You have already submitted a verification attempt, so you don't need to reverify.")}
```

Comment:
You have already submitted your verification information. You will see a message on your dashboard when your identity has been verified (usually within 1-2 days).